src/display/annotation_storage.js

```diff
@@ -282,6 +312,13 @@ class PrintAnnotationStorage extends AnnotationStorage {
   get serializable() {
     return this.#serializable;
   }
+
+  get modifiedIds() {
+    return {
+      ids: new Set(),
+      hash: "",
+    };
```

Comment:
Since this cannot change, let's avoid creating multiple `Set`s during printing.
```suggestion
    return shadow(this, "modifiedIds", {
      ids: new Set(),
      hash: "",
    });
```